utils/SummarySpan.md

```diff
@@ -0,0 +1,164 @@
+# Objective
+A request for a typical Service goes through a cluster of Load Balancers
+before it is processed by the server. Sometimes there may be more stages in
+between. Each stage introduces some processing delay. There is also a network
+delay between these stages. These delays contribute to overall response time of
+the request. 
+
+
+From a perspective of a typical Service customer, a response time of a
+query is the time elapsed between the first Load Balancers (LB) of the provider
```

Comment:
Mention that we refer to L7 LBs here, for L4 LBs we are not going to do too much now. Also for client side LB is different as well.

Reply:
Added a note.